controls/overlay.rb

```diff
@@ -1012,7 +1216,7 @@
     roles_sql = 'SELECT r.rolname FROM pg_catalog.pg_roles r;'
 
     describe sql.query(roles_sql, [pg_db]) do
-      its('lines') { should cmp authorized_roles}
+      its('lines.sort') { should cmp authorized_roles.sort}
```

Comment:
we could use the `be_in` matcher here right? and not have to worry about he element order